lib/game/board.ex

```diff
@@ -56,15 +56,14 @@ defmodule Game.Board do
     trunc(:math.sqrt(Enum.count(board)))
   end
 
-  @spec calculate_turn(map()) :: String.t()
+  @spec calculate_turn(map()) :: :player_one | :player_two
   def calculate_turn(board) do
     board_values = Map.values(board)
-    x_count = Enum.count(board_values, fn symbol -> symbol == "X" end)
-    o_count = Enum.count(board_values, fn symbol -> symbol == "O" end)
-    if x_count == o_count do
-      "X"
+    empty_count = Enum.count(board_values, fn symbol -> symbol == :empty end)
+    if rem(empty_count, 2) == rem(board_dimensions(board), 2) do
+      :player_one
     else
-      "O"
+      :player_two
```

Comment:
I think there is an even simpler solution here, but you don't need to implement it now. I think you could determine whether it is player one or player two's turn based on whether the number of non-empty spaces is even/odd.